### DISCUSSION TITLE
Did the slack community move here?

### DISCUSSION THREAD

**Discussion by leanjunio** [vercel/vercel#4641] [General] [answered]
I was looking around for a discord community chat or help or someting. Has everythnig moved here?

**Comment by markozxuu** [comment, marked as answer]
@leanjunio  Hi 👋  Vercel does not have a community chat (Discord or Slack). They moved to `GitHub Discussions` because it is much more organized on the topic of answering specific questions.

if you have any questions related to `Next.js` you can check their [discussion](https://github.com/vercel/next.js/discussions) system and if the questions are about something related to the Vercel platform feel free to create another question 🙏

**Reply by leanjunio** [reply]
cool, thanks!

**Comment by lukewhitmore** [comment]
Hi there .. is https://vercel-community.slack.com/ not for general membership?
